artefacts.md

```diff
@@ -0,0 +1,19 @@
+# Description of artefacts provided by CI/CD system
```

Comment:
we should mention the following for completeness (I know previous release notes did not!):

is compression enabled ? and which libraries

also is namespace enabled, is strict includes enabled (I've asked Lucian for a description of what exactly it means)